packages/GA4Client/src/initialization.js

```diff
@@ -50,6 +50,7 @@ var initialization = {
         gtag('config', measurementId, configSettings);
 
         gtag('get', measurementId, 'client_id', function (clientId) {
+            // TODO: Does GTAG Hash the clientId somehow?
```

Comment:
Can you share why this matters?  Seems like a GA4 SDK internal implementation detail that we don't need to test, but maybe I'm not thinking about something.